### PR TITLE
GtkPatch.AboutDialog.vala Remove using libs

### PR DESCRIPTION
**Pull request by danrabbit**
* Remove `using Gtk` and namespace Gtk explicitly
* Remove `using Gdk` and namespace Gdk explicitly
* One true brace style
* Space before parentheses

### CODE REVIEW

File: lib/GtkPatch/AboutDialog.vala [hunk 1]
```diff
@@ -246,91 +242,90 @@ public class Granite.GtkPatch.AboutDialog : Gtk.Dialog
...
+        set_default_response (Gtk.ResponseType.CANCEL);
 
 
         Granite.Widgets.Utils.set_theming (this, STYLESHEET, null,
                                            Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
         // Set the default containers
-        Box content_area = (Box)get_content_area();
-        Box action_area = (Box)get_action_area();
+        var content_area = (Gtk.Box)get_content_area ();
```
Comment (Philip-Scott) [inline lib/GtkPatch/AboutDialog.vala]: If you're already touching the line, you should put a space between the cast and the function :)

File: lib/GtkPatch/AboutDialog.vala [hunk 2]
```diff
@@ -246,91 +242,90 @@ public class Granite.GtkPatch.AboutDialog : Gtk.Dialog
...
         authors_label.halign = Gtk.Align.START;
-        authors_label.set_line_wrap(true);
+        authors_label.set_line_wrap (true);
 
-        artists_label = new Label("");
-        artists_label.set_selectable(true);
+        artists_label = new Gtk.Label ("");
+        artists_label.set_selectable (true);
         artists_label.halign = Gtk.Align.START;
         artists_label.set_line_wrap(true);
```
Comment (Philip-Scott) [inline lib/GtkPatch/AboutDialog.vala]: Do you wanna add more spaces between parenthesis?

Comment (danrabbit) [inline lib/GtkPatch/AboutDialog.vala]: I was trying to only change lines I was already touching. I'll come back for the other ones, didn't want the diff to be too big

**Comment by Philip-Scott** [comment]
Should be got for merging. But i'll still wait for the Cody approval since it is granite

**Pull request merged**